tests/gdx-tests-android/AndroidManifest.xml

```diff
@@ -4,7 +4,7 @@
       android:versionCode="1"
       android:versionName="1.0"
       android:installLocation="preferExternal">
-	<uses-sdk android:minSdkVersion="8" android:targetSdkVersion="19"/>
+	<uses-sdk android:minSdkVersion="9" android:targetSdkVersion="19"/>
```

Comment:
Good thing that the support libraries that are supposed to provide compatibility back to SDKs 4 and 7 now "require" a minimum of SDK 9. Good job Google.